test/common/netlib.py

```diff
@@ -132,9 +132,9 @@ def cleanupDevs():
         self.allow_journal_messages("pcp-archive: no such metric: network.interface.* Unknown metric name",
                                     "direct: instance name lookup failed: network.*")
 
-    def get_iface(self, m, mac):
-        def getit():
-            path = m.execute(f"grep -li '{mac}' /sys/class/net/*/address")
+    def get_iface(self, mac: str) -> str:
```

Comment:
Might be nice to turn `m` into an optional kwarg defaulting to `self.machine`... I can imagine that someone else theoretically wants to use this for other machines at some point.

Reply:
Well we generally don't do this, and nothing used it